empiric-package/empiric/core/config.py

```diff
@@ -25,7 +25,7 @@ class IConfig(Protocol):
 
 class BaseConfig(IConfig):
     ADMIN_ADDRESS = 0x0704CC0F2749637A0345D108AC9CD597BB33CCF7F477978D52E236830812CD98
-    DEFAULT_AGGREGATION_MODE = AggregationMode.MEDIAN
+    DEFAULT_AGGREGATION_MODE = AggregationMode.MEDIAN.value
```

Comment:
Interesting, I'm not too familiar with this syntax - why do we need the `.value`? Seems prettier without it

Reply:
it's an enum, you need to access the value